### ISSUE TITLE
Data Mapper icon bootstrap/ng-bootstrap tooltip overflow issue

### ISSUE THREAD

**Issue by gashcrumb** [syndesisio/syndesis#2314]
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[x] Bug report  
[ ] Documentation issue or request
</code></pre>

## The problem

When in the data mapper *sometimes* when hovering over buttons or icons the UI opens a tooltip, however when the tooltip element is triggered it causes the page to overflow, shifting all the elements on the page slightly.  This causes the tooltip to close, the page elements stop overflowing, but then because the user is still hovering the tooltip appears again.

Basically this means the tooltip flashes wildly and elements rapidly vibrate back and forth :-)

It's something that doesn't happen consistently but when it does you can sometimes click on the "show mappings" icon to make it stop.

## Expected behavior

Hover tooltips shouldn't cause elements to overflow and scrollbars to appear.

**Comment by gashcrumb** [comment]
Checking this on a fresh build of master there's some differences I'm seeing with how the tooltips behave.  In tp4 they always appear top-oriented, however currently in master I see that the tooltips auto-positioning is working, so tooltips that were partially overflowing the page no longer seem to.

**Comment by gashcrumb** [comment]
@michael-coker @igarashitm was the data mapper component updated recently?  Trying to reproduce this particular issue and noticed the mapper is generally looking fantastic :-)

**Comment by igarashitm** [comment]
Yep, AtlasMap 1.34.1 which contains @michael-coker 's fix for #1988 is now used in Syndesis ;-)

**Comment by gashcrumb** [comment]
Awesome, yeah, I think this also fixed the behavior I've been seeing folks hit with tp4, great!
